applications/DEMApplication/custom_utilities/inlet.cpp

```diff
@@ -540,7 +552,7 @@ namespace Kratos {
 
             if (number_of_particles_to_insert) {
                 //randomizing mesh
-                srand(/*time(NULL)* */r_modelpart.GetProcessInfo()[TIME_STEPS]);
+                std::mt19937 random_generator(r_modelpart.GetProcessInfo()[TIME_STEPS]);
```

Comment:
so you decided to keep this?

Reply:
Yes, I still think it is an improvement, though it should be done in many other places.